ui/modal/modalWalletUnlock/view.jsx

```diff
@@ -67,7 +66,7 @@ class ModalWalletUnlock extends React.PureComponent<Props, State> {
         confirmButtonLabel={__('Unlock')}
         abortButtonLabel={__('Exit')}
         onConfirmed={() => unlockWallet(password)}
-        onAborted={quit}
+        onAborted={closeModal}
```

Comment:
@saltrafael, I think the original behavior is "do nothing for web", since `quit` will do nothing for web. We might want to retain that behavior instead of closing the modal, as there might be sync issues if user doesn't unlock their wallet.

Can check with @tzarebczan , I believe he has an encrypted wallet for testing on web.